sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineCustomImage.java

```diff
@@ -19,19 +19,39 @@
 @Fluent
 public interface VirtualMachineCustomImage
     extends GroupableResource<ComputeManager, ImageInner>, Refreshable<VirtualMachineCustomImage> {
-    /** @return true if this image was created by capturing a virtual machine */
+    /**
+     * Whether this image was created by capturing a virtual machine.
+     *
+     * @return true if this image was created by capturing a virtual machine
+     */
     boolean isCreatedFromVirtualMachine();
 
-    /** @return the hyper v Generation */
+    /**
+     * Gets the hyper v Generation.
+     *
+     * @return the hyper v Generation
+     */
     HyperVGenerationTypes hyperVGeneration();
 
-    /** @return ID of the virtual machine if this image was created by capturing that virtual machine */
+    /**
+     * Gets ID of the virtual machine.
+     *
+     * @return ID of the virtual machine if this image was created by capturing that virtual machine
```

Comment:
Use the full description in the `@return` in description. "Gets ID of the virtual machine." is not complete.